src/App.jsx

```diff
@@ -14,57 +15,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood === '' ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              setSelectedGood('');
+            }}
+          />
+        </h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
```

Comment:
1. add a react `key`
2. move `selectedGood === good` to a variable
```suggestion
          {goods.map(good => {
            const isActive = selectedGood === good;
            
            <tr
              key={good}
```

Reply:
> 1. add a react `key`
> 2. move `selectedGood === good` to a variable

![image](https://github.com/mate-academy/react_goods-selector-js/assets/149102466/42d403b2-a74c-4635-8798-9a0c0587cfa0)

Reply:
Feel free to ask for help in the chat

Reply:
@pasha1932 hello!
Sorry, it was my mistake. Forgot to add `return`
It should looks like 
```
          {goods.map(good => {
            const isActive = selectedGood === good;
            
            return (
              <tr
                key={good}
```